mobile/apps/photos/lib/utils/bg_task_utils.dart

```diff
@@ -95,11 +94,20 @@ class BgTaskUtils {
           requiresStorageNotLow: false,
           requiresDeviceIdle: false,
         ),
-        existingWorkPolicy: workmanager.ExistingWorkPolicy.append,
+        existingWorkPolicy: workmanager.ExistingPeriodicWorkPolicy.update,
         backoffPolicy: workmanager.BackoffPolicy.linear,
         backoffPolicyDelay: const Duration(minutes: 15),
       );
       $.info("WorkManager configured");
+
+      // Check if task is scheduled (Android only)
+      if (Platform.isAndroid) {
+        final isScheduled = await workmanager.Workmanager()
+            .isScheduledByUniqueName(backgroundTaskIdentifier);
+        $.info(
+          "Periodic task scheduled status: $isScheduled for $backgroundTaskIdentifier",
+        );
```

Comment:
It might be better to log warning if it's not scheduled instead of logging info here. Easier to parse and less noise in the logs